README.md

```diff
@@ -57,22 +55,18 @@ heroku ps:scale web=1
 ### Add the Heroku log drain using the app slug and password created above.
 
 ```
-heroku drains:add https://<your-app-slug>:<password>@<this-log-drain-app-slug>.herokuapp.com/ --app <your-app-slug>
+heroku drains:add https://<username>:<password>@<this-log-drain-app-slug>.herokuapp.com?app=<your-app-slug> --app <your-app-slug>
```

Comment:
Why change that?

Reply:
Previous the basic auth username was used as the app name, gin requires having a predefined list of [basic auth accounts](https://godoc.org/github.com/gin-gonic/gin#BasicAuth) which I don't want to have as I want any app that passes auth to be able to submit logs.